egs/tedlium/s5/local/chain/run_tdnn.sh

```diff
@@ -27,8 +27,7 @@ affix=
 stage=0 # After running the entire script once, you can set stage=12 to tune the neural net only.
 train_stage=-10
 get_egs_stage=-10
-speed_perturb=true
-dir=exp/chain/tdnn  # Note: _sp will get added to this if $speed_perturb == true.
+dir=exp/chain/tdnn$affix
```

Comment:
$affix here will be empty and gets a value only after the parse_options.sh line even if the user supplied --affix option.
So the affix should be added to $dir after the parse_options.sh line.